### ISSUE TITLE
Genesis Pier Solar CD Audio

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/BizHawk#239]
```
Pier Solar is a cart game, but has a CD that will play CD audio if available.

GPGX supports this, but our frontend does not at the moment.  Implementing this 
capability is mostly about answering questions about the frontend and multi-rom 
gameplaying.



```

Original issue reported on code.google.com by `goyu...@gmail.com` on 4 Aug 2014 at 11:29

**Comment by nattthebear** [comment]
@adelikat Here's the xml file I'm trying to load, but it does not work.

```xml
<BizHawk-XMLGame System="GEN" Name="Pier Solar">
  <LoadAssets>
    <Asset FileName=".\Pier Solar.md" />
    <Asset FileName=".\Pier Solar.ccd" />
  </LoadAssets>
</BizHawk-XMLGame>
```

**Comment by nattthebear** [comment]
Fixed in 14c117e92e891e05f9dbf46bff261e8 and it sounds great.
